Makefile

```diff
@@ -82,6 +82,26 @@ minor-release:
 major-release:
 	./scripts/create_release_pr -m
 
+# Integration tests
+
+.PHONY: integration-tests
+integration-tests: $(PELORUS_VENV)
+	. ${PELORUS_VENV}/bin/activate && \
+	./scripts/run-integration-tests
+
+# Unit tests
+.PHONY: unit-tests
+unit-tests: $(PELORUS_VENV)
```

Comment:
Can we add the following comment to explain the CLI flags?

```
# -r: show extra test summaRy: (a)ll except passed, (p)assed
# because using (A)ll includes stdout
# -m filters out integration tests
```

Reply:
done